lib/content-services/src/lib/document-list/services/document-list.service.ts

```diff
@@ -34,14 +34,7 @@ export class DocumentListService implements DocumentListLoader {
 
     static ROOT_ID = '-root-';
 
-    _nodesApi: NodesApi;
-    get nodes(): NodesApi {
```

Comment:
We can't explicitly instantiate the API client classes and should just rely on injected services